### DISCUSSION TITLE
Is there a way to have motion only detect motion in specific time periods?

### DISCUSSION THREAD

**Discussion by chegancasb** [Motion-Project/motion#1765] [Q&A] [answered]
I wanted to restrict the motion detection to certain time periods. I wanted to avoid detect events when we are leaving/returning home for work and school (happens on very specific time windows on weekdays). Is there a way to do this with motion still available?

Could not find anything related with that on documentation.

Thanks

**Comment by tosiara** [comment, marked as answer]
There is no built-in feature that could enable/disable motion detection based on the schedule. But you can use an external script or cron to send API request. Check webcontrol_interface option before using it

Example: `curl http://localhost:8080/0/detection/pause`

**Comment by pataquets** [comment]
@chegancasb Issue gardening. If the solution works, please close discussion as solved and mark @tosiara's reply as answer.

**Comment by lbr38** [comment]
Hello, If I may, I can recommend using motion-UI (https://github.com/lbr38/motion-UI). It is a web interface for motion that I developed.

It allows you to define an autostart & stop of the motion service based on time slots, or based on the presence of a network device on the local network (for example, a smartphone). I personally use this feature to automatically disable motion when I come home and enable it when I leave.
